tailcfg/tailcfg.go

```diff
@@ -1596,3 +1604,18 @@ type SSHAction struct {
 	// value.
 	HoldAndDelegate string `json:"holdAndDelegate,omitempty"`
 }
+
+// ServerMetaResponse is the JSON response to a request to /meta?v=<n>
+// over HTTPS (regular TLS) to the Tailscale control plane server,
+// where <n> is the client's CurrentMapRequestVersion.
+type ServerMetaResponse struct {
+	// PublicNacl specifies the server's original NaCl-based machine key.
```

Comment:
NaCl is a library with several cryptographic primitives. NaCl I believe calls this crypto_box, which is sometimes called naclbox (but maybe not outside of Go?).

Reply:
How about just LegacyPublicKey and PublicKey and we can remove the Legacy one later?